benchmark/client/h2_client.go

```diff
@@ -275,9 +275,13 @@ func main() {
 		return
 	}
 
+	// Disable certificate security checks
 	tlsConfig := &tls.Config{
 		InsecureSkipVerify: true,
 	}
+	if client_type == "h3" {
```

Comment:
I think this needs to be done for both types of http protocols.

Also create a flag called insecureSkipVerify and then perform this action under that flag.